src/main/java/soolutions/hometaskapp/core/Amount.java

```diff
@@ -0,0 +1,40 @@
+package soolutions.hometaskapp.core;
```

Comment:
I think to maintain consistency this file should go in an `/Amount` folder. It would also be easier to navigate as "core" doesn't mean anything in this domain (Let me know if it does and I'm just missing something).

Reply:
Maybe "common" or "shared" would be a better name than "core" then? Both of those seem more obvious to me. Also, with the project being it it's infancy there could be some value in deferring these decisions for a bit until we know more about the other objects that take shape.